test/TestFrameworkDependency.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <ItemGroup Condition="'$(ReferenceTypeForTestFramework)'=='' OR '$(ReferenceTypeForTestFramework)'=='Project'">
+    <ProjectReference Include="..\..\src\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(ReferenceTypeForTestFramework)'=='NuGet'">
+    <PackageReference Include="Microsoft.ML" Version="1.5.0-preview-28305-3" />
```

Comment:
You could use the shared property for the package version here as I suggested on my previous comment.

Reply:
same as pervious

---
In reply to: [343778600](https://github.com/dotnet/machinelearning/pull/4444#discussion_r343778600) [](ancestors = 343778600)